WKElementaryDark.css

```diff
@@ -1479,6 +1602,18 @@
     text-shadow: none;
   }
 
+  /* 
+   * Make the meaning MUCH more visible when exposed
+   */
+  .subject-section__subsection
+    .subject-section__meanings:first-child
+    .subject-section__meanings-items,
+  .reading-with-audio__reading {
+    color: var(--ED-text-hl);
+    font-size: x-large;
+    font-weight: bold;
+  }
+
```

Comment:
I will probably opt to use filters here down the line. Its a great addition, but something like this should probably be user tweakable.